src/dash-table/syntax-tree/lexeme/expression.ts

```diff
@@ -0,0 +1,59 @@
+import isNumeric from 'fast-isnumeric';
+import * as R from 'ramda';
+
+import { LexemeType, IUnboundedLexeme } from 'core/syntax-tree/lexicon';
+import { ISyntaxTree } from 'core/syntax-tree/syntaxer';
+import operand from './operand';
+
+const STRING_REGEX = /^(('([^'\\]|\\'|\\)+')|("([^"\\]|\\"|\\)+")|(`([^`\\]|\\`|\\)+`))/;
```

Comment:
hmm looking again at this regex - `('([^'\\]|\\'|\\)+')` will match `'stuff\'` ie anything starting with a simple quote and ending with an escaped quote. Perhaps the group should be `('([^'\\]|\\.)+')`? Similarly for the other quote types.